docs/site/native-backend.md

```diff
@@ -0,0 +1,123 @@
+---
+layout: site
+title: Native Backend guide for SystemDS
+---
+<!--
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+This guide covers the Native BLAS optimizations and software setup for using SystemDS `native` mode.
+
+- [Requirements](#requirements)
+  - [Hardware](#hardware)
+  - [Software](#software)
+    - [INTEL MKL](#intel-mkl)
+    - [OpenBLAS](#openblas)
+- [Native BLAS setup](#native-blas-setup)
+  - [Add library to the System path](#add-library-to-the-system-path)
+  - [Enable Native BLAS in SystemDS](#enable-native-blas-in-systemds)
+  - [Troubleshooting](#troubleshooting)
+
+
+# Native BLAS mode
+
+SystemDS implements all the matrix operations in Java. This simplifies deployment especially in
+a distributed environment.
+
+In Some cases (such as Deep Neural Networks), to take advantage of native BLAS instead of SystemDS
+internal Java library for performing single node operations such as matrix multiplication, convolution etc.
+
+By default, SystemDS will first attempt to use Intel MKL (if installed), and then OpenBLAS (if installed).
+If none of the libraries are available, SystemDS falls back to its internal java library.
+
+> Note: Current SystemDS version only supported on **Linux** platform.
```

Comment:
There is Windows and Linux support on the x86-64 platform for OpenBLAS, MKL and CUDA.